src/app/shared/stache/modules/breadcrumbs/breadcrumbs.component.spec.ts

```diff
@@ -53,13 +54,12 @@ describe('StacheBreadcrumbsComponent', () => {
 
   beforeEach(() => {
     TestBed.configureTestingModule({
-      imports: [
-        StacheNavModule
-      ],
       declarations: [
+        StacheNavComponent,
```

Comment:
We can't use the NavModule in any of the code run by skyux, and it needs the component in order to run the test...